constants/vars.js

```diff
@@ -1,7 +1,7 @@
 export const N_A = 'N/A'
 export const TOTAL_LABEL = 'Total'
 export const TOTALS = 'Totals'
-export const DISALLOWED_STUDIES = ['combined', 'files']
+export const DISALLOWED_STUDIES = ['combined', 'files', 'ProNET']
```

Comment:
? How is this related? Random bug fix or something?

Reply:
The ProNet site is made up and this ensures that this data is not included in the charts results

Reply:
So unrelated bug fix?